scipy/stats/_morestats.py

```diff
@@ -680,7 +680,7 @@ def probplot(x, sparams=(), dist='norm', fit=True, plot=None, rvalue=False):
         return osm, osr
 
 
-@xp_capabilities(np_only=True)
+@xp_capabilities(out_of_scope=True)
```

Comment:
Marked by mistake since it's mixed in with all the plot functions.
```suggestion
@xp_capabilities(np_only=True)
```